classes/publication/PKPPublication.php

```diff
@@ -119,9 +85,8 @@ public function getFullTitles(string $format = 'text')
      */
     public function getLocalizedTitle($preferredLocale = null, string $format = 'text')
     {
-        $usedLocale = null;
-        $title = $this->getLocalizedData('title', $preferredLocale, $usedLocale);
-        $prefix = $this->getData('prefix', $usedLocale);
+        $title = $this->getLocalizedData('title', $preferredLocale);
+        $prefix = $this->getLocalizedData('prefix', $preferredLocale);
```

Comment:
Unfortunately, I would leave getLocalizedData in this class as it was, i.e. using also usedLocale :-\ because: the title is required to be entered in the publication locale but prefix is an optional metadata, so it can happen that we can prefix in a different locale than the tile which will create an incorrect title.

Reply:
@bozana thanks for spotting and clearing the usage of `selectedLocale` as I was not aware of it. We can centralized that behaviour also and I have updated the `DataObject::getLocalizedData` implementation as that behaviour can be later useful to other places. But if any more modified implementation required, then each class have to implement their own version of `getLocalizedData` or `getLocalePrecedence` method as per need .